### PR TITLE
Cleanup find element docs, add browser.find() info

### PR DESCRIPTION
**Pull request by jsfehler**

### CODE REVIEW

File: docs/find-elements.rst [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+    browser = Browser()
+    browser.set_find_strategy('css')
+
+    elems = browser.find('#btn_13')
+
+This will stay in effect for the entire life cycle of your script, unless
+explicitly changed again.
+
+Although ``browser.find()`` has a less implicit naming structure, it's also less verbose.
```
Comment (fsouza) [inline docs/find-elements.rst:75]: explicit? (or implicit without "less"?)

File: docs/find-elements.rst [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+    browser = Browser()
+
+    all_divs = browser.find_by_tag("div")
+    child_elements = all_divs.first.find_by_css(".item")
+
+Find Links
+==========
+
+To target only links on a page, the ``links`` namespace provides find 4 methods.
```
Comment (fsouza) [inline docs/find-elements.rst:100]:
```suggestion
To target only links on a page, the ``links`` namespace provides 4 find methods.
```

**Review by fsouza — APPROVED** [review]
Couple of comments, non-blocking.

File: docs/find-elements.rst (hunk 1 above)
Comment (jsfehler) [reply docs/find-elements.rst]: :+1: Fixed

File: docs/find-elements.rst (hunk 2 above)
Comment (jsfehler) [reply docs/find-elements.rst]: :+1: Fixed

**Pull request merged**